src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns a special character based on the state of the Task.
+     * @return a Tick if the task is marked as done or a cross if not.
+     */
+    public String getStatusIcon() {
+        return isDone? "\u2713" : "\u2718";
+    }
+
+    /**
+     * Marks the Task as done.
+     */
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * Returns a String to be written into saved data.
+     * @return a String that contains the necessary information in the correct format.
+     */
+    public String storageForm() {
+        return "";
+    }
```

Comment:
i think it is not encouraged to put dummy value like empty string as a return value cuz it might cause confusion. Maybe we can declare Task as an abstract class because there is no need to instantiate Task instance. So, in this case, we can make storageForm abstract and force the subclasses to override this method to carry out the specified behavior.